### DISCUSSION TITLE
Why does it take so long for me to get access ?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#142313] [GitHub Education] [answered]
### Hi everyone,

I seem to be having a problem I have not been granted access when being accepted for a while now can anyone help?

**Comment by davevad93** [comment, marked as answer]
Hi @lawrencemillard ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @lawrencemillard was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by ghost** [reply]
ok thx also how do I get GitHub pro?

**Reply by davevad93** [reply]
Hey @lawrencemillard you're welcome, when your benefits are available you will also get GitHub Pro, you'll know because a `PRO` badge will be added to your profile.

**Comment by ghost** [comment]
Argh

**Comment by ghost** [comment]
@davevad93 It seems its been over 80 hours now and I still do not have access any tips to get access?

**Reply by davevad93** [reply]
Hey @lawrencemillard , as stated above, if your status is still green you have to wait because GitHub no longer provides manual reviews.

Hint: once your profile displays the `PRO` badge, that's when you'll be able to get full access to your benefits.

![0](https://github.com/user-attachments/assets/da94e62a-837c-4e41-ba28-a72306b8f21b)

If by then you are still unable to access them, you should contact support by opening a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

**Reply by JohnDelatto** [reply]
> Hey @lawrencemillard , as stated above, if your status is still green you have to wait because GitHub no longer provides manual reviews.
> 
> Hint: once your profile displays the `PRO` badge, that's when you'll be able to get full access to your benefits.
> 
> ![0](https://private-user-images.githubusercontent.com/94308234/379853099-da94e62a-837c-4e41-ba28-a72306b8f21b.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAwOTc5ODQsIm5iZiI6MTczMDA5NzY4NCwicGF0aCI6Ii85NDMwODIzNC8zNzk4NTMwOTktZGE5NGU2MmEtODM3Yy00ZTQxLWJhMjgtYTcyMzA2YjhmMjFiLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI4VDA2NDEyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJlYmY4ODJkZTI1ZTdkY2E3ZjY4Y2UxMjYxNmI2YzYyMGFlZjU2NzIyM2I5ZTE5Yjk4ZTVjYTI0NWUxMTlhZjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.7J4XRFyYpoX2VzVVmUsYhTjBufXXP37i5kAShhsPfmM)
> 
> If by then you are still unable to access them, you should contact support by opening a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

I need the team to restore my account status

Kindly please approve my request
